src/tests/ftest/osa/osa_online_reintegration.py

```diff
@@ -222,22 +222,37 @@ def run_online_reintegration_test(self, num_pool):
             self.pool.display_pool_daos_space("Pool space: Beginning")
             pver_begin = self.get_pool_version()
             self.log.info("Pool Version at the beginning %s", pver_begin)
-            output = self.dmg_command.pool_exclude(self.pool.uuid,
-                                                   rank, t_string)
+            if server_boot is False:
+                output = self.dmg_command.pool_exclude(self.pool.uuid,
+                                                       rank, t_string)
+            else:
+                output = self.dmg_commmand.system_stop(self.pool.uuid,
```

Comment:
(pylint-no-member) Instance of 'OSAOnlineReintegration' has no 'dmg_commmand' member